sqldelight-sample/src/main/java/com/example/sqldelight/hockey/data/Player.java

```diff
@@ -27,9 +28,16 @@ public Player create(long id, String firstName, String lastName, int number, Lon
     }
   }, DATE_ADAPTER, SHOOTS_ADAPTER, POSITION_ADAPTER);
 
-  public static final class Marshal extends PlayerMarshal<Marshal> {
-    public Marshal() {
-      super(DATE_ADAPTER, SHOOTS_ADAPTER, POSITION_ADAPTER);
-    }
+  public static final RowMapper<ForTeam> FOR_TEAM_MAPPER = FACTORY.for_teamMapper(
+      new For_teamCreator<Player, Team, ForTeam>() {
+        @Override public ForTeam create(Player player, Team team) {
+          return new AutoValue_Player_ForTeam(player, team);
```

Comment:
Once the Android Gradle plugin 2.2.0 ships we can probably switch to Jack and use method references here without all the Retrolambda hoops.